svgdigitizer/svgplot.py

```diff
@@ -204,22 +220,182 @@ class SVGPlot:
     def __init__(
         self,
         svg,
-        xlabel=None,
-        ylabel=None,
+        page_orientation=PageOrientation.PORTRAIT,
         sampling_interval=None,
         curve=None,
         algorithm="axis-aligned",
     ):
         self.svg = svg
-        self.xlabel = xlabel or "x"
-        self.ylabel = ylabel or "y"
+        self.page_orientation = page_orientation
         self.sampling_interval = sampling_interval
         self._curve = curve
         self._algorithm = algorithm
 
     @property
     @cache
-    def axis_labels(self):
+    def xlabel(self):
+        r"""
+        Return the label of the x axis.
+
+        EXAMPLES::
+
+            >>> from svgdigitizer.svg import SVG
+            >>> from io import StringIO
+            >>> svg = SVG(StringIO(r'''
+            ... <svg>
+            ...   <g>
+            ...     <path d="M 0 200 L 0 100" />
+            ...     <text x="0" y="200">E1: 0 cm</text>
+            ...   </g>
+            ...   <g>
+            ...     <path d="M 100 200 L 100 100" />
+            ...     <text x="100" y="200">E2: 1cm</text>
+            ...   </g>
+            ...   <g>
+            ...     <path d="M -100 100 L 0 100" />
+            ...     <text x="-100" y="100">j1: 0</text>
+            ...   </g>
+            ...   <g>
+            ...     <path d="M -100 0 L 0 0" />
+            ...     <text x="-100" y="0">j2: 1 A</text>
+            ...   </g>
+            ... </svg>'''))
+            >>> plot = SVGPlot(svg)
+            >>> plot.xlabel
+            'E'
+
+            >>> plot = SVGPlot(svg, page_orientation=PageOrientation.LANDSCAPE)
+            >>> plot.xlabel
+            'j'
+
+        """
+
```

Comment:
Does black really force these empty lines on us? That's weird.

Reply:
No it does not, checked it right now.

> > So it seems that my locally installed black thinks differently than the one of the github actions.
> 
> Are you using conda? If so, you can force the exact same version in your local installation.
> 
> We should probably set a version number in environment.yml to make it clear which version of black we are using.

Now, I have setup a dedicated environment from the provided yaml. It seems the work now.